controller/array_action/array_mediator_svc.py

```diff
@@ -470,24 +472,22 @@ def _stop_and_delete_fcmap(self, fcmap_id):
 
     def _delete_all_fcmaps_as_source_if_exist(self, snapshot_name):
         snapshot_as_source_fcmaps = self._get_fcmaps(snapshot_name, ENDPOINT_TYPE_SOURCE)
-        unfinished_fcmaps = [fcmap.name for fcmap in snapshot_as_source_fcmaps
```

Comment:
why was this "moved"?
IMO, for fcmaps as source, if there is still even 1 unfinished fcamp, there is no point in deleting **any** fcmap.

Reply:
I think its true for ExpectedSnapshotButFoundVolumeError but for ObjectIsStillInUseError we are going to delete the object after the "use" is over anyway..

Reply:
and _delete_all_fcmaps_as_source_if_exist is not in use anymore (removed now)

Reply:
I don't agree that the "use" is necessarily over, because the unfinished fcmap might still be important for idempotency info of the volume being created